lib/matplotlib/path.py

```diff
@@ -95,6 +95,9 @@ class Path:
                              CURVE4: 3,
                              CLOSEPOLY: 1}
 
+    # A list of path codes for which the vertices are ignored
+    IGNORED_CODES = [STOP, CLOSEPOLY]
```

Comment:
Maybe
```suggestion
    CODES_WITH_IGNORED_VERTICES = [STOP, CLOSEPOLY]
```
A bit lengthy but the name is clearer.